src/scss/notice/_regular.scss

```diff
@@ -21,15 +21,15 @@
 			&-main {
 				@extend %o-message-alert-notice-content-main;
 			}
+
+			&-highlight {
+				@include oTypographyBold('sans');
```

Comment:
Minor: Could this be a placeholder to avoid duplication css?

Reply:
:+1: